mkdirp/mkdirp.ts

```diff
@@ -0,0 +1,24 @@
+import { ErrorKind, FileInfo, chmod, lstat, mkdir, platform } from 'deno'
+
+const PATH_SEPARATOR: string = platform.os === 'win' ? '\\' : '/'
+
+export default async function mkdirp (path: string, mode?: number) : Promise<void> {
```

Comment:
Std library prefers to use named exports over default ones, can you remove the `default`?